### DISCUSSION TITLE
BlackHole 2-channel installed, can I uninstall HomeBrew?

### DISCUSSION THREAD

**Discussion by Timbo368** [ExistentialAudio/BlackHole#241] [Q&A] [answered]
I installed HomeBrew and everything that goes with it. And have BlackHole 2 channel working successfully. Can I just uninstall HomeBrew now? I have never needed it before BlackHole so I don't foresee needing it in the future either.

**Comment by devinroth** [comment, marked as answer]
Sure. BlackHole doesn't rely HomeBrew.
